lua/kommentary/util.lua

```diff
@@ -33,6 +42,24 @@ function M.insert_at_beginning(line, prefix)
     return string.sub(line, 0, start_index-1) .. prefix .. string.sub(line, start_index, #line)
 end
 
+--[[--
+Insert prefix at index.
+@tparam string line String to which the prefix will be prepended
+@tparam string prefix Prefix that will be inserted at index
+@tparam int index Where to insert the prefix
+@treturn string String with prefix before first non-whitespace character
+]]
+function M.insert_at_index(line, prefix, index)
+    -- If the line is empty, just return the prefix with any whitespace stipped
+    if line == nil or line == '' then
```

Comment:
```suggestion
    if M.is_empty(line) then
```
We can use the new is_empty function here.